src/Amazon.Lambda.Tools/LambdaPackager.cs

```diff
@@ -661,5 +663,36 @@ private static void BundleWithZipCLI(string zipCLI, string zipArchivePath, strin
                 }
             }
         }
+
+        internal static void LogDotNetCore20DeprecationMessageIfNecessary(IToolLogger logger, string targetFramework)
```

Comment:
Rename the method to LogDeprecationMessageIfNecessary as this method will probably be reused when .NET Core 1.0 is deprecated.

Reply:
changed (made plural in case they overlap)